JM/CF/Scripts/1_Core/CommunityFramework/CF_PackedByte.c

```diff
@@ -0,0 +1,117 @@
+class CF_PackedByte
```

Comment:
What is a packed byte? Does this mean bit stuffing as in multiple values in one byte?

Reply:
Linked list of `CF_Byte`, optimized reading and writing.